app/components/entryReview/commentConstants.ts

```diff
@@ -6,10 +6,10 @@ export enum EntryAction {
     UNCONTROL = 4,
 }
 
-export const commentTypeToTextMap: { [id: number]: string } = {
-    0: 'commented',
-    1: 'verified',
-    2: 'unverified',
-    3: 'controlled',
-    4: 'uncontrolled',
+export const commentTypeToTextMap: { [id: string]: string } = {
+    COMMENT: 'commented',
+    VERIFY: 'verified',
+    UNVERIFY: 'unverified',
+    CONTROL: 'controlled',
+    UNCONTROL: 'uncontrolled',
 };
```

Comment:
I think we can add a more strict type to the `id`. Maybe there is an enum field in `#generated/types`.